src/Cli.Tests/EndToEndTests.cs

```diff
@@ -374,7 +377,7 @@ public Task TestConfigGeneratedAfterAddingEntityWithSourceWithDefaultType()
     public void TestUpdateEntity()
     {
         string[] initArgs = { "init", "-c", TEST_RUNTIME_CONFIG_FILE, "--database-type",
-                              "mssql", "--connection-string", "localhost:5000" };
```

Comment:
Did you intend to change the instances where `"localhost:5000"` proceeded `"--connection-string"`? Unclear at first glance why replacing localhost with your sample connection string doesn't break tests.

Reply:
I get where `"--connection-string", "testconnectionstring",` is changed to `"--connection-string", "testconnectionstring",` but I don't understand why
`"--connection-string", "localhost:5000"` was changed to `"--connection-string", "testconnectionstring",`